src/julia_internal.h

```diff
@@ -1187,6 +1187,14 @@ struct typemap_intersection_env {
 };
 int jl_typemap_intersection_visitor(jl_typemap_t *a, int offs, struct typemap_intersection_env *closure);
 
+// -- simplevector.c -- //
+
+// For codegen only.
+JL_DLLEXPORT size_t (jl_svec_len)(jl_svec_t *t) JL_NOTSAFEPOINT;
+JL_DLLEXPORT int8_t jl_svec_isassigned(jl_svec_t *t JL_PROPAGATES_ROOT, ssize_t i) JL_NOTSAFEPOINT;
+JL_DLLEXPORT jl_value_t *jl_svec_ref(jl_svec_t *t JL_PROPAGATES_ROOT, ssize_t i);
```

Comment:
Why are these changes in this PR?  Did a rebase/merge go wrong?

Reply:
after cleaning up my local repo, `git push --set-upstream origin krc/cholesky` did the job. This PR has now only the final commit.